packages/button-theme/src/Modal.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { applyTheme, StyleConfig } from '@button-inc/component-library/Modal';
+import styles from './styles';
+
+const modalStyles = {
```

Comment:
Normally I'd just use the default style object until we have real designs, but without any styling it's impossible to tell what the modal looks like, so I grabbed these ones from the bcgov modal and changed the colours